lib/CMakeLists.txt

```diff
@@ -64,27 +70,51 @@ if (SWIFT_SWIFT_PARSER)
       )
     endif()
 
-    add_custom_command(
-      OUTPUT "${SWIFT_HOST_LIBRARIES_DEST_DIR}/${sharedlib}"
-      DEPENDS "${SWIFT_SYNTAX_LIBRARIES_BUILD_DIR}/${sharedlib}"
-      COMMAND ${CMAKE_COMMAND} -E copy_if_different ${SWIFT_SYNTAX_LIBRARIES_BUILD_DIR}/${sharedlib} ${SWIFT_HOST_LIBRARIES_DEST_DIR}/${sharedlib}
-      ${add_origin_rpath}
-    )
-
-    add_custom_target(copy_swiftSyntaxLibrary_${sharedlib}
-      DEPENDS "${SWIFT_HOST_LIBRARIES_DEST_DIR}/${sharedlib}"
-      COMMENT "Copying ${sharedlib}"
-    )
-
-    swift_install_in_component(
-      PROGRAMS "${SWIFT_HOST_LIBRARIES_DEST_DIR}/${sharedlib}"
-      DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/host"
-      COMPONENT compiler
-    )
+    if(CMAKE_SYSTEM_NAME MATCHES Windows)
```

Comment:
Could we just set `SWIFT_HOST_LIBRARIES_DEST_DIR` and `SWIFT_SYNTAX_LIBRARIES_BUILD_DIR` depending on the platform above?

Would have to add a new variable for the install destination, but that seems reasonable.

Reply:
We use both paths so I don't think that we can.  Could you share more about what you were thinking?

Reply:
Where does Windows use `SWIFT_HOST_LIBRARIES_DEST_DIR` and `SWIFT_SYNTAX_LIBRARIES_BUILD_DIR`? All I'm saying is to do:
```
if (CMAKE_SYSTEM_NAME MATCHES Windows)
  set(SWIFT_SYNTAX_LIBRARIES_BUILD_DIR
      "${SWIFT_PATH_TO_EARLYSWIFTSYNTAX_BUILD_DIR}/lib/swift/host")
  set(SWIFT_HOST_LIBRARIES_DEST_DIR
      "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/swift/host")
  set(SWIFT_HOST_LIBRARIES_INSTALL_DIR
      "bin")
else()
  set(SWIFT_SYNTAX_LIBRARIES_BUILD_DIR
      "${SWIFT_PATH_TO_EARLYSWIFTSYNTAX_BUILD_DIR}/bin")
  set(SWIFT_HOST_LIBRARIES_DEST_DIR
      "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
  set(SWIFT_HOST_LIBRARIES_INSTALL_DIR
      "lib${LLVM_LIBDIR_SUFFIX}/swift/host")
end()
```

And then remove the added `if` here + change `swift_install_in_component` to use `SWIFT_HOST_LIBRARIES_INSTALL_DIR`.